api/end_device_services.proto

```diff
@@ -34,8 +34,8 @@ service EndDeviceRegistry {
     };
   };
 
-  // Get the end device with the given identifiers, selecting the fields specified
-  // in the field mask.
+  // Get the end device with the given identifiers. To return additional fields,
+  // specify them in the field mask.
```

Comment:
This is true for almost all RPCs, so if we write it here, we should write it everywhere. I'm fine with leaving it out of here, since the field mask is already described in the request message.